crates/ruma-client-api/src/discovery/get_capabilities.rs

```diff
@@ -292,6 +301,26 @@ impl Default for ThirdPartyIdChangesCapability {
     }
 }
 
+/// Information about the `m.get_login_token` capability.
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+pub struct GetLoginTokenCapability {
+    /// `true` if the user can request a login token, `false` otherwise.
```

Comment:
```suggestion
    /// Whether the user can request a login token.
```

?

Reply:
I was copying the style of other docs in this file, but your version is definitely better.